content/conscious-coliving.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Conscious Coliving: A Way of Living to Thrive"
+description: Living together in conscious communities is a concrete practical action to enable personal, collective and societal flourishing.
+---
+![[conscious-coliving-page-header.jpg]]
+
+### Living together in conscious communities is a concrete practical action to enable personal, collective and societal flourishing.
+
+[[conscious-coliving-course|New to conscious coliving? Start here.]]
+
+## What is Conscious Coliving? 
+
+There’s not really an established term yet for the kind of way of living that we imagine and seek to pioneer. We've come to call it "conscious coliving" reflecting the fusion of two related practices: conscious living and coliving.
+
+It is the practice of living in community, in any format, and dedicating oneself to conscious living and embodiment. The spectrum of conscious coliving environments is broad and can be anything from three friends living in a house together, to an entire village. 
+
+[[what-is-conscious-coliving|For a more detailed definition read our article: What is conscious coliving?]]
+
+## Learn About Conscious Coliving
+
+If you are curious about conscious coliving and want to implement it into your life, our **free email courses** are a great place to begin. 
+
+### Out Now
+
+[[conscious-coliving-course|Conscious Coliving 101:Learn How to Live a More Connected Life]]
+
+### Coming Soon (Pre-Apply Now!)
+
+
+
+### The Online Community for Conscious Coliving
+
+Join 100+ other curious conscious colivers to learn, share and connect in a simple, easygoing way. No thrills, just a chat channel! 
+
+[Join Now](https://chat.whatsapp.com/DycZceubrTzI1DHKHZr5sw)
+
+
+## Why is Conscious Coliving Important?
+
+At the core of Life Itself's Theory of Change, is that for us to step into the new paradigm we must create seeds for the [[second-renaissance|Second Renaissance]]. These seeds are cultivated from physical spaces where people can connect deeply and innovate social, spiritual and practical elements for the next paradigm.
+
+Conscious coliving is therefore fundamental to seeding the Second Renaissance. By living together we are actively and consciously working on our personal, interpersonal and collective development. 
+
+To gain more context on why conscious coliving is important read our article about deliberately developmental space (AKA Conscious Coliving Spaces): [[deliberately-developmental-spaces-a-key-to-addressing-the-metacrisis|Deliberately Developmental Spaces: A Key to Addressing the Metacrisis]]
+
+## Our Approach to Conscious Coliving: Research, Education, Practice, Support
```

Comment:
This sounds more like stuff that would go on an intiatives page where we explain do people who really care about the intiative what we are doing.

That said it could be here too ...

Reply:
I think the purpose of this landing page was also to link together all of our offers and show a cohesion between them, but may be a better way of doing this. I think there should be mention of all these ways we approach exploring CC on this page though